[Docs] Add known 8.6 metric visualization upgrade issue (#152847)
elastic/kibana: docs/CHANGELOG.asciidoc

This known issue only impacts the Observability Rules page. To work around this issue, create APM latency threshold rules in the APM Alerts and Rules dialog. See {kibana-ref}/apm-alerts.html[Alerts and rules] for detailed instructions.
====

[[known-issue-152847]]

> .Metric visualization progress bars get turned off on upgrade

Reviewer: ```suggestion
.Progress bars are hidden in Lens metric visualizations
```

[%collapsible]
====
*Details* +
When upgrading to 8.6, progress bars on existing Lens metric visualizations get hidden.

> To show the progress bar again, open the visualization in the Lens editor. Click the dimension button for the primary metric to open the dimension editor. Then, under "Supporting visualization," select "Bar."

Reviewer: ```suggestion
To display progress bars in existing *Lens* metric visualizations:
. Open the main menu, then click *Visualize Library*.
. From the *Visualize Library* list, click the metric visualization. 
. In *Lens*, click the *Primary metric* field in the layer pane.
. In the *Type* field, select *Bar*.
. Click *Close*.
```
